plugins/woocommerce/changelog/update-53954-payments-menu-notification-badge

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: tweak
+Comment: We reduce the contexts in which the Payments main menu item gets a red notice badge to only incentives.
+
+
```

Comment:
Nitpick: Extra empty line.

Reply:
That line is controlled by the changelog generator. It is as intended.